### PR TITLE
lesson_1_4

### PR DESCRIPTION
**Pull request by mvstoyan**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
+import TodoList from "./TodoList";
 
 function App() {
-  const [newTodo, setNewTodo] = React.useState();
+
+  function addTodo(newTodo) {
+    setTodoList([...todoList, newTodo]);
+  }
+
+  const [todoList, setTodoList] = React.useState([]);
```
Comment (XeniyaDob) [inline src/App.js]: By convention, it is best to declare the useState hook right at the beginning of the function.

**Review by XeniyaDob — APPROVED** [review]
Fantastic work, Marina! Everything works well. 

I suggest you consider this. Separate your work into several commits, how it is separated in the assignments. 

For example, in the Lesson 1.4 
- First commit is `Move Todo List into State`
- Second is `Control "Add Todo" Input`
etc

File: src/App.js (hunk 1 above)
Comment (mvstoyan) [reply src/App.js]: app

**Comment by mvstoyan** [comment]
> Fantastic work, Marina! Everything works well.
> 
> I suggest you consider this. Separate your work into several commits, how it is separated in the assignments.
> 
> For example, in the Lesson 1.4
> 
> * First commit is `Move Todo List into State`
> * Second is `Control "Add Todo" Input`
>   etc

commits
Thank you for the review. I have split the commits as per the tutorial 

https://dev.to/timmouskhelichvili/how-to-split-a-git-commit-into-multiple-ones-3g6f. 

I chose to split them directly rather than creating a branch from the current one.

**Comment by XeniyaDob** [comment]
> > Fantastic work, Marina! Everything works well.
> > I suggest you consider this. Separate your work into several commits, how it is separated in the assignments.
> > For example, in the Lesson 1.4
> > 
> > * First commit is `Move Todo List into State`
> > * Second is `Control "Add Todo" Input`
> >   etc
> 
> commits Thank you for the review. I have split the commits as per the tutorial
> https://dev.to/timmouskhelichvili/how-to-split-a-git-commit-into-multiple-ones-3g6f.
> 
> I chose to split them directly rather than creating a branch from the current one.

Yes, no rush.  I know you already submitted a bunch of homework. So, you do not need to redo them, just use this method in the future :blush:

**Comment by mvstoyan** [comment]
> > > Fantastic work, Marina! Everything works well.
> > > I suggest you consider this. Separate your work into several commits, how it is separated in the assignments.
> > > For example, in the Lesson 1.4
> > > 
> > > * First commit is `Move Todo List into State`
> > > * Second is `Control "Add Todo" Input`
> > >   etc
> > 
> > 
> > commits Thank you for the review. I have split the commits as per the tutorial
> > https://dev.to/timmouskhelichvili/how-to-split-a-git-commit-into-multiple-ones-3g6f.
> > I chose to split them directly rather than creating a branch from the current one.
> 
> Yes, no rush. I know you already submitted a bunch of homework. So, you do not need to redo them, just use this method in the future 😊

I haven't actually tried splitting commits using commands yet, so this was a good situation to try and learn how to handle such situations. So everything is fine🙂. I apologize, I'm moving forward, because I'm taking two courses simultaneously. I hope this doesn't put any pressure on you, you can review all my homework when its will be comfortable for you😌.

**Pull request merged**